### PR TITLE
update example in README according to tests

### PR DESCRIPTION
**Pull request by Brasilikum**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,6 @@ And set this formatter as the new logging formatter:
 
 ```go
 log.Infof("Log messages are now sent to Graylog (udp://%s)", graylogAddr) // Give a hint why logs are empty
-log.AddHook(graylog.NewGraylogHook(graylogAddr, "api", map[string]interface{}{})) // set graylogAddr accordingly
+log.Hooks.Add(graylog.NewGraylogHook(graylogAddr, "api", map[string]interface{}{})) // set graylogAddr accordingly
```
Comment (gravis) [inline README.md]: Why this change? The logrus readme has the same syntax, I can't any reference to `Hooks.Add`.

Comment (gravis) [inline README.md]: hmm, we have another occurence in our Readme, I guess it's a copy/paste from something existing. Do you mind changing this https://github.com/gemnasium/logrus-graylog-hook/blame/master/README.md#L27 as well? And I will merge this PR.
Thanks!

File: README.md [hunk 2]
```diff
@@ -18,12 +18,12 @@ The hook must be configured with:
...
     "log/syslog"
     "github.com/Sirupsen/logrus"
-    "github.com/gemnasium/logrus-hooks/graylog
+    "github.com/gemnasium/logrus-hooks/graylog"
     )
 
 func main() {
     log := logrus.New()
-    hook, err := log.AddHook(graylog.NewGraylogHook("<graylog_ip>:<graylog_port>", "some_facility", map[string]interface{}{"foo": "bar"}))
+    hook := graylog.NewGraylogHook("<graylog_ip>:<graylog_port>", "some_facility", map[string]interface{}{"foo": "bar"})
```
Comment (gravis) [inline README.md]: What do you think of replacing `map[string]interface{}{"foo": "bar"})` with something like`map[string]interface{}{"this": "will be logged every time"})` instead?

File: README.md (hunk 1 above)
Comment (Brasilikum) [inline README.md]: I already changed both occurences, right? [Hooks.Add Reference](https://github.com/Sirupsen/logrus/blob/master/hooks.go#L18)

Comment (gravis) [inline README.md]: ok!

**Comment by gravis** [comment]
thanks a lot

**Comment by Brasilikum** [comment]
:-)

**Pull request merged**